client/client.go

```diff
@@ -106,6 +112,25 @@ func NewCfClient(sdkKey string, options ...ConfigOption) (*CfClient, error) {
 	}
 
 	client.start()
+	if config.waitForInitialized {
+		var initErr error
```

Comment:
So the isIinitialized will still iterate 30 times, with a 2s sleep.
I'm wondering if we should let the user pass in a timeout that they are prepared to wait for.

The reason being, some apps don't need flag values on startup and can happily wait for it to init.
Others might need some flag evaluations in order to startup.   They might be prepared to wait 20s, but not 60s.   A scenario here could be if its a k8 pod, and the startup/readyness probes fail after 30s and restart the pod.  

In this case the customer might want the pod to restart, equally they might be happy to timeout after 20s, let the pod startup and accept the defaults.

Reply:
Discussed offline with you and we were both happy that this can be achieved by setting max auth retries.